### PR TITLE
[enterprise-4.13] OSDOCS-5611: add ssh key location

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.13

Issue: https://issues.redhat.com/browse/OSDOCS-5611

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OSDOCS-5611/release_notes/ocp-4-13-release-notes.html#ocp-4-13-notable-technical-changes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
4dcbc79..14deba2 add `[discrete]` to entry

**Comment by jldohmann** [comment]
@mike-nguyen PTAL for QE ack. This PR adds the SSH key location change to the OCP 4.13 RN

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -450,6 +450,11 @@ To manage the cloud controller manager and cloud node manager deployments and li
 
 For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_cluster-operators-ref[Cluster Cloud Controller Manager Operator] entry in the _Platform Operators reference_.
 
+[discrete]
+[id="ocp-4-13-rhcos-ssh-key-location"]
+==== Change in SSH key location
+{product-title} {product-version} introduces a {op-system-base} 9.2 based {op-system}. Previously, SSH keys were located in `/home/core/.ssh/authorized_keys` on {op-system}. Now, on {op-system-base} 9.2 based {op-system}, SSH keys are located in `/home/core/.ssh/authorized_keys.d/ignition`.
```
Comment (tmalove) [inline release_notes/ocp-4-13-release-notes.adoc:456]:
```suggestion
{product-title} {product-version} introduces a {op-system-base} 9.2 based {op-system}. Before this update, SSH keys were located in `/home/core/.ssh/authorized_keys` on {op-system}. With this update, on {op-system-base} 9.2 based {op-system}, SSH keys are located in `/home/core/.ssh/authorized_keys.d/ignition`.
```
Hi @jldohmann , consider these suggestions.

**Review by tmalove — APPROVED** [review]

**Comment by tmalove** [comment]
/remove-label peer-review-in-progress
/label peer-review-done

Just a couple of suggestions, otherwise looks good!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (jldohmann) [reply release_notes/ocp-4-13-release-notes.adoc]: Thanks, Tami. I've implemented these suggestions 🙂

**Comment by mike-nguyen** [comment]
LGTM

**Pull request merged**